build-system-tests/templates/components/react/next/App.js

```diff
@@ -1,78 +1,70 @@
-initializeInAppMessaging();
-
-export default function Home() {
-  useEffect(() => {
-    // sync remote in-app messages
-    syncMessages();
-  }, []);
-
+const Home = () => {
   return (
-    <>
-      <InAppMessagingProvider>
-        <InAppMessageDisplay />
-        <Text>In-App Messaging Example</Text>
-      </InAppMessagingProvider>
-      <Authenticator>
-        {({ signOut, user = { username: '' } }) => (
-          <main>
-            <h1>Hello {user.username}</h1>
-            <button onClick={signOut}>Sign out</button>
-          </main>
-        )}
-      </Authenticator>
-      <AccountSettings.ChangePassword onSuccess={() => {}} />
-      <AccountSettings.DeleteUser onSuccess={() => {}} />
-      <StorageManager
-        acceptedFileTypes={['image/*']}
-        accessLevel="guest"
-        maxFileCount={1}
-        isResumable
-      />
-      <StorageManager
-        acceptedFileTypes={['image/*']}
-        path="public/"
-        maxFileCount={1}
-        isResumable
-      />
-      <FileUploader
-        acceptedFileTypes={['image/*']}
-        accessLevel="guest"
-        maxFileCount={1}
-        isResumable
-      />
-      <FileUploader
-        acceptedFileTypes={['image/*']}
-        path="public/"
-        maxFileCount={1}
-        isResumable
-      />
-      <MapView
-        initialViewState={{
-          latitude: 37.8,
-          longitude: -122.4,
-          zoom: 14,
-        }}
-      >
-        <LocationSearch />
-      </MapView>
-    </>
+    <Authenticator>
+      {({ signOut, user = { username: '' } }) => (
+        <main>
+          <h1>Hello {user.username}</h1>
+          <button onClick={signOut}>Sign out</button>
+          <FaceLivenessDetector
+            sessionId="123"
+            region="us-east-1"
+            onAnalysisComplete={async () => {}}
+          />
+          <AccountSettings.ChangePassword onSuccess={() => {}} />
+          <AccountSettings.DeleteUser onSuccess={() => {}} />
+          <StorageManager
+            acceptedFileTypes={['image/*']}
+            accessLevel="guest"
+            maxFileCount={1}
+            isResumable
+          />
+          <StorageManager
+            acceptedFileTypes={['image/*']}
+            path="public/"
+            maxFileCount={1}
+            isResumable
+          />
+          <FileUploader
+            acceptedFileTypes={['image/*']}
+            accessLevel="guest"
+            maxFileCount={1}
+            isResumable
+          />
+          <FileUploader
+            acceptedFileTypes={['image/*']}
+            path="public/"
+            maxFileCount={1}
+            isResumable
+          />
+          <MapView
+            initialViewState={{
+              latitude: 37.8,
+              longitude: -122.4,
+              zoom: 14,
+            }}
+          >
+            <LocationSearch />
+          </MapView>
+        </main>
+      )}
+    </Authenticator>
   );
-}
+};
+
+export default dynamic(() => Promise.resolve(Home), {
```

Comment:
what does this do?

Reply:
This was to resolve [this](https://github.com/aws-amplify/amplify-ui/actions/runs/11281713272/job/31377552683) build error (and it's modeled after our liveness `next` examples) but I can't replicate the error anymore so took it out